config/base/catalogd/crd/bases/olm.operatorframework.io_clustercatalogs.yaml

```diff
@@ -0,0 +1,441 @@
+---
```

Comment:
See: https://github.com/operator-framework/operator-controller/tree/2278c8a97e0bc84fc88aa35bdc18df551a90c448/config/base/crd/bases

Should the operator-controller CRD not be under base/operator-cotroller//crd/bases as well?

Reply:
The ClusterExtension CRD _is_ under `base/operator-controller/crd/bases:
https://github.com/operator-framework/operator-controller/pull/1790/files#diff-86654bcc3610d7644076892e7bc003a1af8040ecb7e4ac233883d0cf563c8561